guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,114 @@
+[id="registering-a-host-to-project-using-the-global-registration-template_{context}"]
+= Registering a Host to {ProjectName} Using the Global Registration Template
+
+Use the following procedure to register a host to {ProjectNameX}.
+
+With this registration method you can register hosts to {Project} by generating a `curl` command on {Project} and running this command on hosts.
+This method uses the global registration template that gives you complete control over the process of the host registration.
+
+ifeval::["{build}" != "satellite"]
+Note that you can extend the parameters by plug-ins.
+For more information, see https://github.com/theforeman/foreman/blob/develop/developer_docs/how_to_create_a_plugin.asciidoc[How to Create a Plugin].
+endif::[]
+
+.Prerequisites
+* The {Project} user that generates the `curl` command must have the `create_hosts` permission.
+* You must have root privileges on the host that you want to register.
+
+.Procedure
+. In the {Project} web UI, navigate to *Hosts* > *Provisioning Templates*, find the *Linux registration default* template and click it.
+
+. Click the *Association* tab.
+
+. Select the operating systems that you want to register.
+
+. Click *Submit*.
+
+. Navigate to *Hosts* > *Operating Systems* and click the operating system that you want to register.
+
+. Click the *Templates* tab.
+
+. From the *Registration template* list, select *Linux registration default*.
```

Comment:
this is the default choice; are you required to describe it like this or can you say: 
From the *Registration template* list, ensure *Linux registration default* is selected.

Reply:
Done.